qtpy/tests/test_qtsql.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import
+
+import pytest
+from qtpy import QtSql
+
+def test_qtsql():
+    """Test the qtpy.QtSql namespace"""
+    assert QtSql.QSqlDatabase is not None
+    # assert QtSql.QSqlDriverCreator is not None
+    assert QtSql.QSqlDriverCreatorBase is not None
+    assert QtSql.QSqlDriver is not None
+    #assert QtSql.QSqlDriverPlugin is not None
```

Comment:
You could move this comment out lines to another test skipped in PYSIDE2

```
from qtpy import PYSIDE2

@pytest.mark.skipif(PYSIDE2, reason="It fails on PySide2")
def test_qtsql_no_pyside():
    assert QtSql.QSqlDriverCreator is not None
    assert QtSql.QSqlDriverPlugin is not None
```

Reply:
I don't think so. It is not part of my PyQt5 environment and the checks told me that it is also not the case for the other three case. Actually for the QSqlDriverCreater class/1st commit the test succeeded for PySide2 because there I excluded PySide2 completely. (Too much copy'n'paste...) But I doubt that it is included there

Reply:
I understood, It's ok to leave if comment out, maybe add a comment explaining that those features aren't yet  present in the Qt API